staging/src/k8s.io/client-go/go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/golang/groupcache v0.0.0-20160516000752-02826c3e7903
 	github.com/golang/protobuf v1.2.0
 	github.com/google/btree v0.0.0-20160524151835-7d79101e329e // indirect
+	github.com/google/go-cmp v0.3.0
```

Comment:
Note that I could "avoid" this direct dependency by continuing to use the deprecated functions from the `k8s.io/apimachinery/pkg/util/diff` package (but that looks like the wrong approach):

```go
func legacyDiff(a, b interface{}) string {
	return cmp.Diff(a, b)
}

// ObjectDiff prints the diff of two go objects and fails if the objects
// contain unhandled unexported fields.
// DEPRECATED: use github.com/google/go-cmp/cmp.Diff
func ObjectDiff(a, b interface{}) string {
	return legacyDiff(a, b)
}
```